model-mesh/dependencies/quickstart.yaml

```diff
@@ -47,6 +47,35 @@ spec:
           configMap:
             name: etcd-scripts
             defaultMode: 0554
+      initContainers: 
+        - name: etcd-secret-creator
+          image: quay.io/openshift/origin-cli
```

Comment:
We need an image from registry.redhat, not quay.

Reply:
updated image